sklearn/metrics/tests/test_classification.py

```diff
@@ -2135,6 +2136,33 @@ def test_hinge_loss_multiclass_missing_labels_with_labels_none():
         hinge_loss(y_true, pred_decision)
 
 
+def test_hinge_loss_multiclass_no_consistent_pred_decision_shape():
+    # test for inconsistency between multiclass problem and pred_decision
+    # argument
+    y_true = np.array([2, 1, 0, 1, 0, 1, 1])
+    pred_decision = np.array([0, 1, 2, 1, 0, 2, 1])
+    error_message = ("The shape of pred_decision cannot be 1d array"
+                     "with a multiclass target. pred_decision shape "
+                     "must be (n_samples, n_classes), that is "
+                     f"({y_true.shape[0]}, {np.unique(y_true).size}). "
+                     f"Got: ({y_true.shape[0]},)")
```

Comment:
The following would make more sense:

```suggestion
                     f"Got: {pred_decision.shape}")
```

Alternatively, just hard code the numbers in the test: " that is (7, 3). Got: (7,).